service/application/src/main/java/com/solace/maas/ep/event/management/agent/command/CommandManager.java

```diff
@@ -204,19 +205,19 @@ private void attachErrorLogToCommand(boolean isTerraformRelatedError, Exception
         }
     }
 
-    private void finalizeAndSendResponse(CommandRequest request) {
-        request.determineStatus();
+    private void finalizeAndSendResponse(CommandRequest requestBO) {
+        requestBO.determineStatus();
         Map<String, String> topicVars = Map.of(
-                "orgId", eventPortalProperties.getOrganizationId(),
+                "orgId", requestBO.getOrgId(),
```

Comment:
No longer grabbing the org ID from the application properties. It now comes from the mop message.